### DISCUSSION TITLE
Identify home backup load vs grid

### DISCUSSION THREAD

**Discussion by anderslew** [kellerza/sunsynk#159] [Q&A] [answered]
Hi,

I'm using this add-on with great succes so far. Mostly to gather info on my inverter, but also to charge my car with excess solar power.

Background:
Right now my house is running of the Backup load port, on a Deye inverter SUN-12K-SG04LP3-EU (with manual switch over in case of issues with inverter). My car charger is permanantly on the Grid side. CT clamps are diretly after the utility meeter, so includes the car charger.

My issue / question:
I'm trying to see the values specifically for the backup load (in my case all load except the car charger). Currently all Load power values I see includes the grid side / car charger.

When looking at solarman business I'm actually able to see the relevant info (see picture). So I assume there is registers specific to read out this data. But I'm not sure how to find them. So I hope someone can help a bit? :-)

On the below screenshot from Solarman Business, I have highlighted the relevant data. The car is charging with 6 AMPs (3 phase), so corresponding to roughly 240 x 3 x 6 = 4320 watt. And it appears that the Load power values I get from the add-on is the "Load phase power A/B/C", and not the "Load power L1/L2/L3". I Think that sensors 650,651,652 must be corresponding to load phase power A/B/C in SOLARMAN business.

Load Power (from definitions3ph.py):
SENSORS += (
    Sensor(653, "Load power", WATT, -1),
    Sensor(650, "Load L1 power", WATT, -1),
    Sensor(651, "Load L2 power", WATT, -1),
    Sensor(652, "Load L3 power", WATT, -1),
)

![image](https://github.com/kellerza/sunsynk/assets/79974408/66426718-5231-45cc-85b0-1266e6697d25)

**Comment by kellerza** [comment]
No real suggestion, but if you are trying to find the issue, try creating your own sensors in mysensors.py & add the now modifier to the sensors you are comparing

**Comment by anderslew** [comment, marked as answer]
Just to update here in case someone else is looking for the same as me. I did find the answer myself. :-)

So something like this:

SENSORS += (
Sensor(643, "UPS Total power", WATT, -1),
Sensor(640, "UPS L1 power", WATT, -1),
Sensor(641, "UPS L2 power", WATT, -1),
Sensor(642, "UPS L3 power", WATT, -1),
)
